### PR TITLE
Implement new PoS Arbitrum and Optimism Goerli testnets #5466

### LINKED ISSUE #5466: Implement new PoS Arbitrum and Optimism Goerli testnets
### IDs:

Optimism Goerli: 420 Currency Symbol: ETH
Arbitrum Goerli: 421613 Currency Symbol: AGOR

### RPC:

Arbitrum Goerli:

https://arbitrum-goerli.infura.io/v3/

Optimism Goerli:

https://optimism-goerli.infura.io/v3/

### Transaction Viewer:

https://goerli-rollup-explorer.arbitrum.io/tx/
https://blockscout.com/optimism/goerli/tx/

### APIs:

https://goerli-rollup-explorer.arbitrum.io/api?
https://blockscout.com/optimism/goerli/api?

Logos would stay the same as current testnet logos.

### Backup RPC nodes:

https://goerli-rollup.arbitrum.io/rpc
https://goerli.optimism.io

This should be enough info for a full implementation. Let me know if you need any more info.

### PR DESCRIPTION
**Pull request by oa-s**
Closes #5466

### CODE REVIEW

**Comment by oa-s** [comment]
@hboon can u please check if this pr works for you, have strange bad access on apps launch, not functional changes only adds a new chain.

**Comment by hboon** [comment]
Will check when I’m back. But try running test suite. It can catch some issues when adding new chains.

**Comment by hboon** [comment]
@oa-s ah, crashes for me too. If I base the commit on top of `48b191cc36264f5287e5509da037c5829c3f083c` (before the creation of `AlphaWalletWeb3`), it doesn't crash. Maybe that helps give you a clue where to look?

**Comment by hboon** [comment]
I forgot to include 843f0691e93387f8b4af9c01c1eec27e32fc0fe9 when basing. So might be related too.

File: modules/AlphaWalletFoundation/AlphaWalletFoundation/Settings/Types/RPCServers.swift [hunk 1]
```diff
@@ -631,6 +639,7 @@ public enum RPCServer: Hashable, CaseIterable {
             case .ioTeX: return "https://babel-api.mainnet.iotex.io"
             case .ioTeXTestnet: return "https://babel-api.testnet.iotex.io"
             case .candle: return "https://rpc.cndlchain.com"
+            case .optimismGoerli: return "https://arbitrum-goerli.infura.io/v3/"
```
Comment (hboon) [inline modules/AlphaWalletFoundation/AlphaWalletFoundation/Settings/Types/RPCServers.swift:642]: I think this should need the key like other Infura-hosted RPC nodes

**Comment by oa-s** [comment]
i have no idea why but removing `import AlphaWalletWeb3` from `RPCServer.swift` fixes crash, magic.

**Comment by hboon** [comment]
> i have no idea why but removing `import AlphaWalletWeb3` from `RPCServer.swift` fixes crash, magic.

😭

**Comment by hboon** [comment]
@oa-s this one's ready?

**Comment by oa-s** [comment]
there is an issue with fetching transactions will check it one more time

**Pull request merged**